From 168d44d169dbb928e4a367c1fdc7a650683007c5 Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Wed, 2 May 2012 22:23:00 +0100
Subject: [PATCH] units: Silence systemd console output on getty start.

This is really just a cosmetic hack until the real upstream fix
is used. It's too invasive to backport right now, so this
fudge will have to do.

This should not go upstream as f2b68789 is a different
(and arguably more correct) way to implement this.
---
 units/getty@.service.m4 |    1 +
 1 file changed, 1 insertion(+)

diff --git a/units/getty@.service.m4 b/units/getty@.service.m4
index a02838d..3254ac5 100644
--- a/units/getty@.service.m4
+++ b/units/getty@.service.m4
@@ -35,6 +35,7 @@ Before=getty.target
 
 [Service]
 Environment=TERM=linux
+ExecStartPre=-/bin/kill -55 1
 ExecStart=-/sbin/agetty %I 38400
 Restart=always
 RestartSec=0
-- 
1.7.10

